csuntechlab/iSTART
# STR-28 -- FE: Create Generic Module Thumbnail for dashboard that can be extended by other modules

# Description
Created reusable module component for dashboard.
  
# Testing
1. Login as **april.feldman** & **cyndee.bellamy**.
2. Ensure the dashboard module accurately represents the one in the mock-up.
3. Check at different responsive sizes and ensure content meets requirements.
**`NOTE: Mock-ups are not finalized and will be missing desktop version; will also update progress circle when the proper package gets added`**
